tools/scale.ts

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env ts-node
+
+/**
+ * Usage:
+ *
+ * Encode storage key:
+ *  ./scale.ts key \
+ *    --pallet System \
+ *    --storage Account \
+ *    --key 0x6Df05cBd5113261Dd0f63fb8ce89FC0e236B844e
+ *
+ * Decode storage item:
+ *  ./scale.ts decode \
+ *    --ws wss://wss.api.moonbase.moonbeam.network \
+ *    --type FrameSystemAccountInfo \
+ *    --value '0x1600...de' \
+ *    --explain
+ *
+ * Encode storage item:
+ * ./scale.ts encode \
+ *    --ws wss://wss.api.moonbase.moonbeam.network \
+ *    --type FrameSystemAccountInfo \
+ *    --value '{  "nonce": "22",
+ *       "consumers": "0",
+ *        ...
+ *      }'
+ */
+
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { u8aToHex, hexToU8a } from "@polkadot/util";
+import { xxhashAsU8a, blake2AsU8a } from "@polkadot/util-crypto";
+import yargs from "yargs";
+import mergeWith from "lodash.mergewith";
+import isObject from "lodash.isobject";
+
+const args = yargs
+  .showHelpOnFail(true)
+  .command(
+    "key",
+    "Compute storage key",
+    {
+      pallet: {
+        type: "string",
+        demandOption: true,
+        alias: "pallet",
```

Comment:
you don't need alias if it is the same key I believe. Alias are if you want a shorter name or just 1 letter

Reply:
ooops that's a mistake